### DISCUSSION TITLE
Able to transfer a repo; Can I transfer the project as well?

### DISCUSSION THREAD

**Discussion by chuck-adaptive** [community/community#36421] [Other Feature Feedback, Questions, & Ideas]
My team and I started on a project in my personal github. We have since moved the code base to a github organization.

I still have the Project board on my personal github, however. I am no longer able to link issues between the project and the new repository.

Is there a way to transfer the Project to the organization? I see no controls to do so or documentation.

Thank you

**Comment by ossanna16** [comment]
Hey there @chuck-adaptive and thank you for asking a good question! I'm glad you found our community 🙂
